source/_integrations/automation.markdown

```diff
@@ -7,6 +7,7 @@ ha_release: 0.7
 ha_quality_scale: internal
 ha_codeowners:
   - '@home-assistant/core'
+ha_iot_class: Local Push
```

Comment:
This is a weird one to classify in the IoT classes.

It isn't a device, it doesn't calculate its state. One could maybe say "Assumed state" since the actual automation entity has a state (but lacks device). The automations one creates and fires, could be considered local push (or cloud push? depends on the service called I guess).

```suggestion
ha_iot_class: ~
```

Suggested to make it `~` (= null / None). What do you think @danimart1991?

Reply:
I look for this "null" value to add to these devices like automation. Thank You @frenck